packages/drawer/src/views/modern/Drawer.tsx

```diff
@@ -339,7 +343,7 @@ export default function Drawer({
             {drawerType !== 'permanent' ? (
               <Overlay
                 progress={progress}
-                onPress={() => toggleDrawer(false)}
+                onPress={() => toggleDrawer(false, true)}
```

Comment:
Hello and thanks for the PR!
Note that I'm not the person who will have a say in merging this PR, but I just wanted to point out that 

`toggleDrawer(false, true)`, or more genetally `someFunction(false, true)` reads pretty badly. I'd rather refactor that into an object with named keys.

Just my 2 cents 🙂

Reply:
Thanks, agreed. I have made the change now.